Algorithm.Python/BasicTemplateCfdAlgorithm.py

```diff
@@ -22,32 +22,32 @@
 
 class BasicTemplateCfdAlgorithm(QCAlgorithm):
 
-    def Initialize(self):
+    def initialize(self):
         '''Initialise the data and resolution required, as well as the cash and start-end dates for your algorithm. All algorithms must initialized.'''
-        self.SetAccountCurrency('EUR')
+        self.set_account_currency('EUR')
 
-        self.SetStartDate(2019, 2, 20)
-        self.SetEndDate(2019, 2, 21)
-        self.SetCash('EUR', 100000)
+        self.set_start_date(2019, 2, 20)
+        self.set_end_date(2019, 2, 21)
+        self.set_cash('EUR', 100000)
 
-        self.symbol = self.AddCfd('DE30EUR').Symbol
+        self.symbol = self.add_cfd('DE30EUR').symbol

```

Comment:
Should use `self._symbol`. No error because `QCAlgorithm.Symbol` is a method, but if the method is used anywhere else in the algo, it will blow up